packages/aws-cdk/test/integ/run-against-dist

```diff
@@ -23,13 +23,15 @@ if [[ ! -f $dist_root/build.json ]]; then
   exit 1
 fi
 
+local local_cli_version="$(node -e "console.log(require('${dist_root}/build.json').version)")"
+
 serve_npm_packages
 
 # Install the CLI and put it on the path
-(cd $npmws && npm install aws-cdk)
+(cd $npmws && npm install aws-cdk@${local_cli_version})
 export PATH=$npmws/node_modules/.bin:$PATH
 
-verify_installed_cli_version
+verify_installed_cli_version ${local_cli_version}
```

Comment:
This is silly now.

Reply:
@rix0rrr Yeah I was contemplating that. Should we just delete this function altogether?

Reply:
Im just gonna leave it for now - seems safer.